### ISSUE TITLE
Maintaining tqdm progress bar callback from keras-tqdm 

### ISSUE THREAD

**Issue by shun-lin** [tensorflow/addons#517]
**Describe the feature and the current behavior/state.**

Keras integration with TQDM progress bars.

The current repository is in https://github.com/bstriner/keras-tqdm and I would like to maintain/update it for tensorflow 2.0 and put it here in the contrib section so more people can easily use it.

**Relevant information**
- Are you willing to contribute it (yes/no): Yes
- Are you willing to maintain it going forward? (yes/no): Yes
- Is there a relevant academic paper? (if so, where): No
- Is there already an implementation in another framework? (if so, where): No 
- Was it part of tf.contrib? (if so, where): No

**Which API type would this fall under (layer, metric, optimizer, etc.)**

callbacks

**Who will benefit with this feature?**

Those who wants to use tqdm to have a prettier progress bar when training models.

**Any other info.**

**Comment by facaiy** [comment]
Looks cool, any thoughts @tensorflow/sig-addons-maintainers

**Comment by seanpmorgan** [comment]
I think its a nice addition (and our first callback). Please be sure to update the callbacks README guidelines if there is anything you find missing from the documentation.

**Comment by shun-lin** [comment]
Hi,

Thank you so much for your comments! I have been busy with work lately but will work on this when I get time!
